Source/WebKit/Shared/API/c/WKURLRequest.cpp

```diff
@@ -61,13 +60,11 @@ WKStringRef WKURLRequestCopyHTTPMethod(WKURLRequestRef requestRef)
 WKURLRequestRef WKURLRequestCopySettingHTTPBody(WKURLRequestRef requestRef, WKDataRef body)
 {
     WebCore::ResourceRequest requestCopy(WebKit::toImpl(requestRef)->resourceRequest());
-    requestCopy.setHTTPBody(WebCore::FormData::create(std::span { WKDataGetBytes(body), WKDataGetSize(body) }));
+    requestCopy.setHTTPBody(WebCore::FormData::create(unsafeForgeSpan(WKDataGetBytes(body), WKDataGetSize(body))));
```

Comment:
Seems like we want a helper function here that gets a `span` for a `WKData`.

Reply:
Ok, adding one. Proposing `WKDataGetSpan()` in `WKData.h`